install/helm/gloo/templates/43-gatewayparameters.yaml

```
@@ -0,0 +1,62 @@
+{{- if .Values.kubeGateway.enabled -}}
+kind: GatewayParameters
+apiVersion: gateway.gloo.solo.io/v1alpha1
+metadata:
+  labels:
+    {{- include "gloo-gateway.constLabels" . | nindent 4 }}
+  name: default
```

can we set this to the name of the GatewayClass?
```suggestion
  name: gloo-gateway
```